lib/express/sgid.js

```diff
@@ -35,11 +35,13 @@ function config(app, { showLoginPage, serviceProvider }) {
 
   app.get(`${PATH_PREFIX}/authorize`, (req, res) => {
     const { redirect_uri: redirectURI, state, nonce } = req.query
+    const scopes = req.query.scope ? req.query.scope : 'openid'
+    console.warn(`Requested scope ${scopes}`)
```

Comment:
Eh what about...
```suggestion
    const scopes = req.query.scope ?? 'openid'
```

Reply:
I tried that at first and eslint complained about it. I suspect our eslint doesn't support es2020 yet.

Reply:
I updated the eslint parser ES version in this PR so I can use the null coalescing operator. Thanks!